ui/app/main/src/components/pages/page-insights.ts

```diff
@@ -110,8 +111,17 @@ const viewerConfig: DataViewerConfig = {
             gridRowStart: "1"
         },
         defaults: [{
-            assetId:"4m4dWZUAV4IepHKc8Wwv85",
+            assetId:"35Q8RvOhBU3boCB6IUW1Dj",
```

Comment:
Guess this is for your testing purposes and will be removed

Reply:
Yes. Robin has put this into his version of the insights page, probably as a working stub, and I connected the KPI cards to it (and therefore I had to fit this in with "my local manager build"). This will be dropped once the localstorage functionality comes in.